### ISSUE TITLE
Error: Required flag "duration" not set ❌

### ISSUE THREAD

**Issue by cloud-66** [josepdcs/kubectl-prof#41]
i want to get only one threaddump ,  tool requires  flag duration.
why ? For my opinion it is kind of bug and should be fixed
```
 kubectl prof pod-7f5c86898c-hp2xn -l java -o threaddump --image=registry/systemtest/kubectl-prof:1.2.5-jvm
```

```
Error: Required flag "duration" not set ❌
```

**Comment by josepdcs** [comment]
I check it

**Comment by josepdcs** [comment]
Fixed [1.2.6](https://github.com/josepdcs/kubectl-prof/releases/tag/1.2.6)
